cookbooks/bcpc-hadoop/attributes/default.rb

```diff
@@ -4,6 +4,8 @@
 #
 #############################################
 
+
+default['yarn']['timelineservice']['hostname']="f-bcpc-vm2.bcpc.example.com"
```

Comment:
This should not be a node attribute but a search using `get_nodes_for()` so that it is found dynamically.